src/lib/pickles/wrap.ml

```diff
@@ -566,7 +566,8 @@ let%test_module "gate finalization" =
 
         let actual_feature_flags =
           { Plonk_types.Features.none_bool with
-            range_check = true
+            range_check0 = true
```

Comment:
Should we also add a test that uses only `range_check0`?

Reply:
Perhaps a 64-bit check alone? Sure, I will add it